deploy/bonfire.example.yaml

```diff
@@ -0,0 +1,17 @@
+# Bonfire deployment configuration
+
+# Defines where to fetch the file that defines application configs
+appsFile:
+  host: gitlab
+  repo: insights-platform/cicd-common
+  path: bonfire_configs/ephemeral_apps.yaml
+
+# (optional) define any apps locally. An app defined here with <name> will override config for app
+# <name> in above fetched config.
+apps:
+- name: provisioning
+  components:
+    - name: service
+      host: local
+      repo: <path_to_service_dir>
```

Comment:
This seems like from a template... just saying. I haven't tried this yet.

Reply:
It is a template, that is why it says `example` in the file name, I might add a make command to customize it.